.github/workflows/build.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         uses: actions/setup-java@v2
         with:
           java-version: ${{ matrix.java }}
-          distribution: 'adopt'
```

Comment:
Any reason to change the `distribution`?

Reply:
I think this is a good move since they have rebranded and recommend people move from Adopt to Termurin.

See - https://blog.adoptopenjdk.net/2021/08/goodbye-adoptopenjdk-hello-adoptium/